orderbook/src/event_updater.rs

```diff
@@ -1,63 +1,21 @@
-use crate::database::Database;
-use anyhow::{Context, Result};
+use crate::database::Postgres;
+use anyhow::Result;
 use contracts::{
-    gpv2_settlement::{self, Event as ContractEvent},
+    gpv2_settlement::{self},
     GPv2Settlement,
 };
-use ethcontract::{dyns::DynWeb3, Event};
-use shared::{
-    event_handling::{BlockNumber, EventHandler, EventStoring},
-    impl_event_retrieving,
-    maintenance::Maintaining,
-};
-use std::ops::RangeInclusive;
+use ethcontract::dyns::DynWeb3;
+use shared::{event_handling::EventHandler, impl_event_retrieving, maintenance::Maintaining};
 use tokio::sync::Mutex;
 
-pub struct EventUpdater(Mutex<EventHandler<DynWeb3, GPv2SettlementContract, Database>>);
-
-#[async_trait::async_trait]
-impl EventStoring<ContractEvent> for Database {
```

Comment:
I can't seem to find where all this went.. Is it somehow inherited from an existing trait somewhere?

Reply:
I moved this implementation into database/events.rs . This is also why in your comment above we now pass a `Postgres` instance directly instead of `Arc<Postgres>`.